testsuite/tests/typing-jkind-bounds/basics.ml

```diff
@@ -1631,3 +1631,18 @@ Error: This expression has type "int t" but an expression was expected of type
        But the kind of int t must be a subkind of value mod portable
          because of the definition of require_portable at line 2, characters 21-57.
 |}]
+
+(*********************************)
+(* Test 15: extensible variants *)
+
+(* The best kind an extensible variant can get is [value] *)
+type extensible : value = ..
+[%%expect{|
+type extensible = ..
+|}]
+
+(* Since the kind is [best], it should normalize away *)
```

Comment:
This doesn't test whether `extensible` normalizes away. I think you'd need something like

```
module M : sig
  type t : immediate with extensible
end = struct
  type t
end
```

Reply:
well, it does in this branch since there's no subsumption :p 

but sure, that's a more robust test anyway